packages/wonder-blocks-dropdown/components/select-opener.js

```diff
@@ -84,14 +84,19 @@ export default class SelectOpener extends React.Component<SelectOpenerProps> {
         ];
 
         return (
-            <ClickableBehavior disabled={disabled} onClick={onClick}>
+            <ClickableBehavior
+                disabled={disabled}
+                onClick={onClick}
+                triggerOnEnter={false}
+            >
                 {(state, handlers) => {
                     const stateStyles = _generateStyles(light, {...state});
                     const {hovered, focused, pressed} = state;
                     return (
                         <StyledButton
                             disabled={disabled}
-                            role="menu"
+                            role="button"
```

Comment:
Hmm, this is an interesting change - just confirming that this makes sense?

Reply:
If this isn't a menu anymore, does the dropdown itself have role="menu"?

Reply:
I did some googling, looks like it should be `listbox`, so I'll change that.